integration-tests/cypress/e2e/project/react-flow/diagram.reactflow.cy.js

```diff
@@ -63,7 +63,11 @@ describe('/projects/:projectId/edit - Diagram', () => {
 
     cy.getByTestId('share').click();
     cy.url().then(($url) => {
-      cy.task('getClipboard').should('eq', $url);
+      cy.window().then((win) => {
+        win.navigator.clipboard.readText().then((text) => {
+          expect(text).to.eq($url);
+        });
+      });
```

Comment:
Would this work too for the Sprotty version? If so, can we get rid of the `clipboardy` dependency?

Reply:
Yes, it's working, I changed it and removed the `clipboardy` dep.